library-best-practices/JLBP-7.md

```diff
@@ -0,0 +1,37 @@
+[JLBP-7] Make breaking transitions easy
+---------------------------------------
+
+When a library makes an in-place breaking change (meaning that the
+Java package is not renamed - see [JLBP-6](JLBP-6.md) for more
+discussion on renaming rules), this can create a lot of work for
+consumers to adapt to the change. If a change is "atomic" (meaning
```

Comment:
I'm not sure the "atomic" section is important since if it's not atomic, it isn't breaking.

Reply:
In the non-atomic process, the second step (removal of deprecated methods) is breaking though...